src/legacy/core_plugins/kibana/public/table_list_view/table_list_view.js

```diff
@@ -472,6 +475,7 @@ TableListViewUi.propTypes = {
   editItem: PropTypes.func.isRequired,
 
   listingLimit: PropTypes.number,
+  hideActionsOnMobile: PropTypes.bool,
```

Comment:
Are you sure you want to hide **all** actions? I know the only action available right now is "edit", but I can foresee more options becoming available down the road like Canvas has:

<img width="807" alt="Screen Shot 2019-04-11 at 11 19 11 AM" src="https://user-images.githubusercontent.com/549577/55969368-a7430280-5c4b-11e9-9253-fdf2fa09c5e5.png">

You may want to consider just hiding the edit ability which may be more of a prop name change that still hides the whole column for now. But it allows the prop to be most specific.

Reply:
Yeah I was thinking about that and basically did it for the easier code implementation, since that component does not yet support any additional actions at all. But I see how one could extend the component later to do it. So I can improve the implementation to basically only hide the edit actions, and the column if no other action exists (which was the main reason why it got easier just hiding the full column all the time).